conformance_tests/sysman/test_sysman_temperature/src/test_sysman_temperature.cpp

```diff
@@ -253,7 +256,8 @@ TEST_F(TEMPERATURE_TEST,
     for (auto temp_handle : temp_handles) {
       ASSERT_NE(nullptr, temp_handle);
       auto temp = lzt::get_temp_state(temp_handle);
-      EXPECT_GT(temp, 0);
+      EXPECT_GE(temp, min_valid_temperature);
```

Comment:
change. temp->temperature or current_temperature.

Reply:
Thanks, Ashwin! Changed it.